sdks/python/setup.py

```diff
@@ -334,11 +334,13 @@ def get_portability_package_data():
             # https://github.com/jupyter/jupyter_client/issues/637
             'jupyter-client>=6.1.11,!=6.1.13,<8.1.1',
             'timeloop>=1.0.2,<2',
+            'nbformat>=5.0.5,<6',
+            'nbconvert>=6.2.0,<8',
           ] + dataframe_dependency,
           'interactive_test': [
             # notebok utils
-            'nbformat>=5.0.5,<6',
-            'nbconvert>=6.2.0,<8',
+            'nbformat>=5.0.5',
+            'nbconvert>=6.2.0',
```

Comment:
```suggestion
            'nbformat>=5.0.5,<6',
            'nbconvert>=6.2.0,<8',
```

One more instance of this!